citesphere/src/test/java/edu/asu/diging/citesphere/core/util/model/impl/CitationHelperTest.java

```diff
@@ -0,0 +1,350 @@
+        AffiliationForm affiliationForm = new AffiliationForm();
+        affiliationForm.setName("Test");
+        affiliationForms.add(affiliationForm);
+        author.setAffiliations(affiliationForms);
+        authors.add(author);
+        form.setAuthors(authors);
+        form.setDateFreetext("2019");
+        form.setVolume("1.0");
+        List<PersonForm> editors = new ArrayList<PersonForm>();
+        PersonForm editor = new PersonForm();
+        editor.setFirstName("editor");
+        editor.setLastName("editor");
+        editors.add(editor);
+        form.setEditors(editors);
+        List<ConceptAssignmentForm> conceptTags = new ArrayList<>();
+        ConceptAssignmentForm tag = new ConceptAssignmentForm();
+        tag.setConceptName("testc");
+        tag.setConceptUri("www.google.com");
+        tag.setConceptId("CON1");
+        tag.setConceptTypeId("CTY1");
+        tag.setConceptTypeName("forum");
+        tag.setConceptTypeUri("www.test.com");
+        conceptTags.add(tag);
+        form.setConceptTags(conceptTags);
+    }
+
+    private void initUpdatedCitation() {
+        updatedCitation = new Citation();
+        updatedCitation.setTitle("updatedtitle");
+        updatedCitation.setShortTitle("updatedShortTitle");
+        updatedCitation.setDateFreetext("2019");
+        updatedCitation.setVolume("1.0");
+        Set<IPerson> editorsSet = new LinkedHashSet<IPerson>();
+        IPerson editor1 = new Person();
+        editor1.setFirstName("editor");
+        editor1.setLastName("editor");
+        editor1.setId("1");
+        editorsSet.add(editor1);
+        updatedCitation.setEditors(editorsSet);
+        Set<IPerson> authorsSet = new LinkedHashSet<IPerson>();
+        IPerson author1 = new Person();
+        author1.setFirstName("first");
+        author1.setLastName("last");
+        author1.setId("2");
+        Set<IAffiliation> affiliations = new LinkedHashSet<IAffiliation>();
+        IAffiliation affiliation = new Affiliation();
+        affiliation.setName("Test");
+        affiliations.add(affiliation);
+        author1.setAffiliations(affiliations);
+        authorsSet.add(author1);
+        updatedCitation.setAuthors(authorsSet);
+        Set<ICitationConceptTag> conceptTagsSet = new LinkedHashSet<ICitationConceptTag>();
+        ICitationConceptTag tag1 = new CitationConceptTag();
+        tag1.setConceptName("testc.com");
+        tag1.setConceptUri("www.google.com");
+        tag1.setLocalConceptId("CON1");
+        tag1.setLocalConceptTypeId("CTY1");
+        tag1.setTypeName("forum");
+        tag1.setTypeUri("www.test.com");
+        conceptTagsSet.add(tag1);
+        updatedCitation.setConceptTags(conceptTagsSet);
+    }
+
+    @Test
+    public void test_updateCitation() {
+        ICitation citation;
+        citation = new Citation();
+        citation.setTitle("title");
+        citation.setShortTitle("shortTitle");
+        citation.setDateFreetext("2019");
+        citation.setVolume("1.0");
```

Comment:
since you're creating the same citation object over and over again, it should be created once in the setup method